### ISSUE TITLE
Refreshing breaks Tiled Gallery display in Block Editor

### ISSUE THREAD

**Issue by kriskarkoski** [Automattic/wp-calypso#30403]
#### Steps to reproduce
1. Starting at URL: Create a new post at wordpress.com/post or /wp-admin/post.php using the Block Editor
2. Add a Tiled Gallery block with images
3. Make sure the draft is saved automatically or manually
4. Refresh and see a "This block has encountered an error and cannot be previewed." message where the block should be. If you proceed to publish the gallery still displays on the front-end

#### What I expected
The gallery to still display in the Block Editor

#### What happened instead
The error "This block has encountered an error and cannot be previewed."

Switching to the code editor the code changes from the following when it works:

```
<!-- wp:jetpack/tiled-gallery {"ids":[175447107,175447106,175447105]} -->
<div class="wp-block-jetpack-tiled-gallery aligncenter is-style-rectangular"><div class="tiled-gallery__gallery"><div class="tiled-gallery__row"><div class="tiled-gallery__col"><figure class="tiled-gallery__item"><img alt="" aria-label="image 1 of 3 in gallery" data-height="640" data-id="175447107" data-link="https://k2testblog.wordpress.com/gallery/attachment/9/" data-url="https://k2testblog.files.wordpress.com/2019/01/9.jpg?w=820" data-width="1920" src="https://k2testblog.files.wordpress.com/2019/01/9.jpg"/></figure></div></div><div class="tiled-gallery__row"><div class="tiled-gallery__col"><figure class="tiled-gallery__item"><img alt="" aria-label="image 2 of 3 in gallery" data-height="640" data-id="175447106" data-link="https://k2testblog.wordpress.com/gallery/attachment/8/" data-url="https://k2testblog.files.wordpress.com/2019/01/8.jpg?w=820" data-width="1920" src="https://k2testblog.files.wordpress.com/2019/01/8.jpg"/></figure></div></div><div class="tiled-gallery__row"><div class="tiled-gallery__col"><figure class="tiled-gallery__item"><img alt="" aria-label="image 3 of 3 in gallery" data-height="640" data-id="175447105" data-link="https://k2testblog.wordpress.com/gallery/attachment/7/" data-url="https://k2testblog.files.wordpress.com/2019/01/7.jpg?w=820" data-width="1920" src="https://k2testblog.files.wordpress.com/2019/01/7.jpg"/></figure></div></div></div></div>
<!-- /wp:jetpack/tiled-gallery -->
```

To this when it doesn't after refreshing:

```
<!-- wp:jetpack/tiled-gallery {"ids":[175447107,175447106,175447105]} -->
<div class="wp-block-jetpack-tiled-gallery aligncenter is-style-rectangular"><div class="tiled-gallery__gallery"><div class="tiled-gallery__row"><div class="tiled-gallery__col"><figure class="tiled-gallery__item"><img alt="" aria-label="image 1 of 3 in gallery" data-id="175447107" data-link="https://k2testblog.wordpress.com/gallery/attachment/9/" src="https://k2testblog.files.wordpress.com/2019/01/9.jpg" /></figure></div></div><div class="tiled-gallery__row"><div class="tiled-gallery__col"><figure class="tiled-gallery__item"><img alt="" aria-label="image 2 of 3 in gallery" data-id="175447106" data-link="https://k2testblog.wordpress.com/gallery/attachment/8/" src="https://k2testblog.files.wordpress.com/2019/01/8.jpg" /></figure></div></div><div class="tiled-gallery__row"><div class="tiled-gallery__col"><figure class="tiled-gallery__item"><img alt="" aria-label="image 3 of 3 in gallery" data-id="175447105" data-link="https://k2testblog.wordpress.com/gallery/attachment/7/" src="https://k2testblog.files.wordpress.com/2019/01/7.jpg" /></figure></div></div></div></div>
<!-- /wp:jetpack/tiled-gallery -->
```

Note the date-width, data-height, and data-url all missing post-refresh. I was not able to reproduce this on an Atomic site using the core Block Editor.

#### Browser / OS version
Chrome 70.0.3538.11
OS X 10

#### Screenshot / Video
Before refreshing

<img width="643" alt="screen shot 2019-01-24 at 3 56 54 pm" src="https://user-images.githubusercontent.com/1900444/51708261-73175880-1ff1-11e9-8c39-82eaf5799a14.png">

After refreshing

<img width="717" alt="screen shot 2019-01-24 at 3 57 07 pm" src="https://user-images.githubusercontent.com/1900444/51708274-7ca0c080-1ff1-11e9-8ea3-783716bc2ffb.png">

#### Context / Source

#manual-testing,  #user-report

**Comment by arunsathiya** [comment]
I tested this a bit, and I am able to reproduce this both on Gutenlypso (Gutenberg on Calypso) and on Calypsoified Gutenberg (`{site}/wp-admin/post-new.php`). 

Tested on a simple site. Admin role.

Same with `Author` role. 

`Contributor` role doesn't seem to have the ability to upload/manage media - so doesn't matter.

<img width="730" alt="screenshot 2019-01-25 at 14 46 35" src="https://user-images.githubusercontent.com/18581859/51736519-4d04bd80-20b0-11e9-948c-dd82d4776f39.png">

On Atomic though, I am not seeing this behaviour. On refreshing the tiled gallery block appears fine. 👍🏽
